api/overview/azure/latest/messaging.eventgrid-readme.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: Azure Event Grid client library for .NET
+description: Provides you an overview of Azure Event Grid library for .NET. 
 keywords: Azure, dotnet, SDK, API, Azure.Messaging.EventGrid, eventgrid
 author: Kishp01
 ms.author: kishp
```

Comment:
Changes to this file will be overwritten on release. Changes to the front matter will also be overwritten. We don't presently support a `description` field at this time so that value would be overwritten on release. 

We do support `ms.service` and you can set that by setting the `MSDocService` field in the [metadata CSV file](https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv#L29). 

Instructions and other information: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/602/Package-Onboarding-step-by-step?anchor=docs-metadata